partner_industry/README.rst

```diff
@@ -0,0 +1,25 @@
+Industry sector on partner
```

Comment:
Hi @jt-xx Thanks for your contribution,
I will check mainly formal aspects of the PR.
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
AFAICS, you miss several parts (headers, proper document title, usage section, bug tracker, issues/roadmap if necessary)
We recommend to include a screenshot as well to help new comers